README.md

```diff
@@ -0,0 +1,18 @@
+# Unchained: A Scala Bitcoin Blockchain Parser
+
+A simple, fast and functional blockchain block parser written in Scala.
+
+This lib is used in the reactive-blockchain project, an Open Source blockchain explorer that processes the blockchain into MongoDB and uses the Play! Framework for exposing the blockchain throught APIs.
+
+** This project is still WIP for now, come back later for more news on when the lib will be published. **
+
+## Example usage
+
```

Comment:
https://github.com/cchantep/sbt-hl-compiler/

Reply:
I'm not a huge fan of automating documentation but I'll look into it in later builds